roles/nodepool/files/etc/nodepool/elements/nodepool/cleanup.d/90-validate-dib-image

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+function reporter () {
+  message="$1"
+  shift
+  echo
+  echo "${message}"
+  for (( i=0; i<${message}; i++ )); do
+    echo -n '-'
+  done
```

Comment:
What are we trying to do with this for loop? Since `$message` is not an integer, I would expect this to fail.